tests/test_samples.py

```diff
@@ -63,6 +69,15 @@ def test_read_getdist():
     mcmc.plot_1d(['x0', 'x1', 'x2', 'x3'])
 
 
+@pytest.mark.xfail('montepython' not in sys.modules,
+                   raises=ImportError,
+                   reason="requires montepython package")
+def test_read_montepython():
+    mcmc = MCMCSamples(root='./tests/example_data/mp')
+    mcmc.plot_2d(['x0', 'x1', 'x2', 'x3'])
+    mcmc.plot_1d(['x0', 'x1', 'x2', 'x3'])
+
```

Comment:
I like the use of this decorator to implement the conditional passing -- not something I've used before.